src/op_analytics/datapipeline/models/code/enriched_traces.py

```diff
@@ -0,0 +1,29 @@
+import duckdb
+
+from op_analytics.datapipeline.models.compute.querybuilder import TemplatedSQLQuery
+from op_analytics.datapipeline.models.compute.registry import register_model
+from op_analytics.datapipeline.models.compute.types import NamedRelations
+
+
+@register_model(
+    input_datasets=["ingestion/traces_v1", "ingestion/transactions_v1", "ingestion/blocks_v1"],
+    expected_outputs=["enriched_traces_v1"],
+    auxiliary_views=[
+        TemplatedSQLQuery(
+            template_name="transaction_fees",
+            context={},
+        ),
+        TemplatedSQLQuery(
+            template_name="enriched_transactions",
+            context={},
+        ),
+        TemplatedSQLQuery(
+            template_name="enriched_traces",
+            context={},
+        ),
+    ],
+)
+def daily_address_summary(duckdb_client: duckdb.DuckDBPyConnection) -> NamedRelations:
```

Comment:
the name of the function will be used by the framework as the name of the model, so you probably want this one to be different.

Reply:
yeah fixing this!